ConsumerApi.Tests.Integration/StepDefinitions/IdentitiesApiStepDefinitions.cs

```diff
@@ -35,14 +29,14 @@ public async Task GivenAChallengeC()
     [When(@"a POST request is sent to the /Identities endpoint with a valid signature on c")]
     public async Task WhenAPOSTRequestIsSentToTheIdentitiesEndpoint()
     {
-        await CreateIdentity();
+        _identityResponse = await CreateIdentity();
     }
 
     [Given(@"an Identity i")]
     public async Task GivenAnIdentityI()
     {
-        await CreateChallenge();
-        await CreateIdentity();
+        _challengeResponse = await CreateChallenge();
+        _identityResponse = await CreateIdentity();
```

Comment:
I know it's not your naming, but:

```suggestion
        _createChallengeResponse = await CreateChallenge();
        _createIdentityResponse = await CreateIdentity();
```